tests/docker_run_tests.rs

```diff
@@ -107,80 +122,218 @@ fn simple_build(path: &str) -> String {
 
     name
 }
+const POSTGRES_IMAGE: &str = "postgres";
+
+struct Network {
+    name: String,
+}
+
+fn attach_container_to_network(network_name: String, container_name: String) {
+    let mut docker_cmd = Command::new("docker");
+
+    docker_cmd
```

Comment:
big ol nit: These docker commands don't need to be set to a variable first. Can just `Command::new("docker").arg("...").....`

Reply:
Borrow checker got angry :(

Reply:
jk